File: FeetToInches.java
import java.util.*;;

public class FeetToInches {

    static final double conversion = 12;

    public static void main(String[] args) {
        double feet_input;
        double Results;

        try ( Scanner s = new Scanner(System.in) ) {
            System.out.print("Enter the Number of Feet: ");
            feet_input = s.nextDouble();
        }

        Results = feetToInches(feet_input);
        System.out.println("Inches: " + Results);

    }

    public static double feetToInches(double feet) {
        return (feet * conversion);
    }

}

File: C++/FeetToInches.cpp
#include <iostream>

using namespace std;

const int inchConversion = 12;

int feetToInches(int feet) {
    return (feet * inchConversion);
}

int main() {

    int ft_input;

    cout << "Enter Number of Feet: ";
    cin >> ft_input;

    int inches = feetToInches(ft_input);
    cout << ft_input << "ft = " << inches << "in";
}